frontend/src/app/util/code_gen.cljs

```diff
@@ -98,9 +125,9 @@
              :text-transform]
    :to-prop {:fill-color "color"}
    :format  {:font-family #(str "'" % "'")
-             :font-style #(str "'" % "'")
+             :font-style #(str %)
```

Comment:
All these `str` should be changed to `dm/str` where dm is `app.common.data.macros` it's more efficient and should be used always for string concatenation.